src/cbc_sdk/enterprise_edr/threat_intelligence.py

```diff
@@ -558,14 +1197,18 @@ def update(self, **kwargs):
                 self.id
             )
 
+        iocs_v2_poked = False
         for key, value in kwargs.items():
             if key in self._info:
                 self._info[key] = value
+            if key == 'iocs_v2':
+                iocs_v2_poked = True
 
         if self.iocs:
             self._iocs = IOC(self._cb, initial_data=self.iocs, report_id=self.id)
-        if self.iocs_v2:
+        if self.iocs_v2 and iocs_v2_poked:
```

Comment:
Don't create a variable when you can check `'iocs_v2' in kwargs`

Reply:
How do you mean? I need to record if the arguments changed iocs_v2, so that we can rebuild the _iocs_v2 array and flag that it no longer needs to be synced.

Reply:
You don't need to create variable to check the kwargs if iocs_v2 exists in the for loop you can check the kwargs directly in the conditional see the following code

```
        for key, value in kwargs.items():
            if key in self._info:
                self._info[key] = value
        
        if self.iocs:
            self._iocs = IOC(self._cb, initial_data=self.iocs, report_id=self.id)
        if self.iocs_v2 and 'iocs_v2' in kwargs:
```

Reply:
OK

Reply:
Thanks!